src/styles/blocks/stars.css

```diff
@@ -0,0 +1,29 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url('../../images/star.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+
+  height: 12px;
+  width: 12px;
+
+  margin: 2px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). In this case, you're setting a margin of 2px on all sides of `.stars__star`, which can lead to margin collapse or inconsistencies. Consider specifying only the necessary margins.